assets/js/blocks/stock-filter/edit.tsx

```diff
@@ -101,14 +86,16 @@ const Edit = ( {
 			{ getInspectorControls() }
```

Comment:
We definitely should. As we agree to use notice instead of the toolbar button, we will rename this component and then add it to all current filter blocks.